freecodecamp.conf.json

```diff
@@ -19,7 +20,7 @@
         "open": true,
         "url": "http://localhost:8080",
         "showLoader": true,
-        "timeout": 20000
+        "timeout": 4000
```

Comment:
is this long enough - this is the loading screen I presume, what happens when it times out?

Reply:
I like it too 👍